docs/AutomationReference.md

```diff
@@ -123,26 +123,46 @@ The `ScannerFactory.CreateScanner` method returns an `IScanner` object.
 
 #### IScanner
 
+##### `ScanAsync`
+The `ScanAsync` method asynchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner. This method should be `await`ed.
+
+###### Parameters
+`IScanner.ScanAsync` accepts two parameters: an instance of `ScanOptions` containing custom settings for this scan (or `null` for default options), and a [`CancellationToken`](https://learn.microsoft.com/en-gb/dotnet/api/system.threading.cancellationtoken?view=netstandard-2.0).
```

Comment:
Can we put these in a table similar to the ScanOptions constructor? That's the norm for our docs